styles/components/alert.scss

```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
```

Comment:
Yea, for docs only. Check https://github.com/dev-onenetworkecommerce/june/pull/21